mlflow/cli/scorers.py

```diff
@@ -59,3 +59,79 @@ def list_scorers(experiment_id: str, output: str) -> None:
         # Table output format
         table = [[name] for name in scorer_names]
         click.echo(_create_table(table, headers=["Scorer Name"]))
+
+
+@commands.command("register-llm-judge")
+@click.option(
+    "--name",
+    "-n",
+    type=click.STRING,
+    required=True,
+    help="Name for the judge scorer",
+)
+@click.option(
+    "--instructions",
+    "-i",
+    type=click.STRING,
+    required=True,
+    help=(
+        "Natural language instructions for evaluation. Must contain at least one "
+        "template variable: {{ inputs }}, {{ outputs }}, {{ expectations }}, or {{ trace }}. "
+        "See the make_judge documentation for the interpretation of these variables."
+    ),
+)
+@click.option(
+    "--model",
+    "-m",
+    type=click.STRING,
+    required=False,
+    help=(
+        "Model identifier to use for evaluation (e.g., 'openai:/gpt-4'). "
+        "If not provided, uses the default model."
+    ),
+)
+@click.option(
+    "--experiment-id",
+    "-x",
+    envvar=MLFLOW_EXPERIMENT_ID.name,
+    type=click.STRING,
+    required=True,
+    help="Experiment ID to register the judge in. Can be set via MLFLOW_EXPERIMENT_ID env var.",
+)
+def register_llm_judge(name: str, instructions: str, model: str | None, experiment_id: str) -> None:
+    """
+    Register an LLM judge scorer in the specified experiment.
+
+    This command creates an LLM judge using natural language instructions and registers
+    it in an experiment for use in evaluation workflows.
+
+    Examples:
+
+    \b
```

Comment:
We'll need rst syntax for code examples for the API docs to build correctly if you want this to be displayed as a code block in the docs (right now it'll show up as plain text)

Reply:
Took a stab based on other comments. PTAL.

Reply:
looks solid!